### PR TITLE
Fixes Issues with the OSS Menu

### LINKED ISSUE #3136: [Bug] /alternative/covid
**Describe the bug**

### /alternative/covid/country

    1. Country command not documented
    2. Autocompletion doesn't work on the countries, ei :
        country F
        country -c F

### /alternative/covid/cases

    it's confusing because `--limit` only apply to the table
    and the table is only displayed when we use raw
    raw shouldn't display chart?

### /alternative/covid/deaths

    same than cases

### /alternative/covid/ov

    same than cases

### /alternative/covid/

    1. bug with : `slopes -d 1`

    ** On entry to DLASCLS parameter number  4 had an illegal value
    Error: SVD did not converge in Linear Least Squares

    Error: 'list' object has no attribute 'columns'

    2. autocompletion not working

    3. export not working

### LINKED ISSUE #3137: [Bug] /alternative/oss
**Describe the bug**

### /alternative/oss/rossidx

    Update doc `-t {stars,forks}` => `-ct {stars,forks}`

### /alternative/oss/rs

    1. This command doesn't work : `rs --help`
    2. Type `rs foo` -> `rs` ; now autocompletion doesn't work
    3. incorrect export written in the web doc
        `{csv,json,xlsx,png,jpg,pdf,svg}` -> `{csv,json,xlsx}`
        https://openbb-finance.github.io/OpenBBTerminal/terminal/alternative/oss/rs/

### /alternative/oss/sh

    same than rs

### /alternative/oss/tr

    1. Need to remove the `US COVID cases` chart from the documentation
        https://openbb-finance.github.io/OpenBBTerminal/terminal/alternative/oss/tr/
    2. This command doesn't work : `tr --raw`

### PR DESCRIPTION
**Pull request by colin99d**
# Description

Fixes #3136 by adding suggestion for countries and adding documentation for the `country` command.
Fixes #3137 by adding docs.

@Chavithra some of the requests are a little confusing for me, lets talk about this.
For some reason rossidx -ct shows choices, but --chart-type does not.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
    - `python terminal.oy alternative/covid/` then type in `country` and watch the pop up.
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/alternative/oss/oss_controller.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def __init__(self, queue: List[str] = None):
                 "--growth": {},
                 "-g": "--growth",
                 "--chart-type": {c: {} for c in ["stars", "forks"]},
-                "-t": "--chart-type",
+                "-ct": "--chart-type",
```
Comment (jmaslek) [inline openbb_terminal/alternative/oss/oss_controller.py:48]: only single letters for single dash commands

**Comment by jmaslek** [comment]
ruh roh.  Another country conversion
```
2022 Nov 03, 12:27 (🦋) /alternative/covid/ $ country -c United Kingdom

The following args couldn't be interpreted: ['Kingdom']
United not a valid selection.
```

**Comment by jmaslek** [comment]
Also do we want the `country` command on hugo? (`about country` gives a 404).  I dont see the need for it, but am open to keeping it

**Comment by colin99d** [comment]
> ruh roh. Another country conversion
> 
> ```
> 2022 Nov 03, 12:27 (🦋) /alternative/covid/ $ country -c United Kingdom
> 
> The following args couldn't be interpreted: ['Kingdom']
> United not a valid selection.
> ```

I thought this was no longer supposed to work? Should we add choices so that people can see their options?

**Comment by jmaslek** [comment]
It was the autocomplete messing with me

Screen Shot 2022-11-04 at 9 15 23 AM

**Review by jmaslek — APPROVED** [review]

**Pull request merged**